zds/member/tests/views/tests_profile.py

```diff
@@ -78,10 +78,20 @@ def test_details_member(self):
         """
         To test details of a member given.
         """
+        # Test staff user profile without content first
```

Comment:
```suggestion
        # Test staff user profile without activity on forums first
```
`content` = `contenu`, c'est ce qu'on utilise pour désigner les tutoriels, articles ou billets, donc utiliser ce terme ici peut prêter à confusion, parce qu'on ne crée pas un contenu, mais de l'activité sur le forum.